decentraland/unity-renderer
# feat: biw project creation

## What does this PR change?

Fix #1400 
Fix #1072 

This PR have the functionality of create a new project from the panel and list the project panel

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-project-creation
2. Create a new project
3. Check that the project is listed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
